### DISCUSSION TITLE
[BUDI-7406] Freezes when selecting from options picker

### DISCUSSION THREAD

**Discussion by tomycastelli** [Budibase/budibase#11593] [Help]
**Checklist**
- [x] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**
- Self
  - Method: docker compose
  - Budibase Version: 2.9.25
  - App Version: 2.9.25

**Describe the bug**
I have an options selector which displays different containers based on the selection made, in Safari  and in the mobile chrome app it works without problems, but in Chrome, firefox and opera in desktop, it freezes when selecting certain options which show a container which has some bindings and js scripting made in some default field values.
The form which is shown works well and uploads data to the db without problems in safari or chrome mobile, but has this problem in chrome and firefox desktop.

**To Reproduce**
1. Go to the page with the title: "Ingreso de operaciones"
2. Click on the options selector called: "Tipo de operación"
3. Select "Cambio de Divisa" or "Cable"
4. The page will freeze and you will have to log in again.

**Expected behavior**
When selecting one of those options, the page will completely freeze and force you to reload and login again

**Screenshots**
<img width="1379" alt="Screenshot 2023-08-18 at 09 43 53" src="https://github.com/Budibase/budibase/assets/69599481/180174b7-4225-4b20-a98b-dbc76341f10b">
  
**App Export**
This is the app
[Financiera-export-1692362330169.tar.gz](https://github.com/Budibase/budibase/files/12379296/Financiera-export-1692362330169.tar.gz)

mail: testing@gmail.com
pass: 6vni488z1r

**Desktop (please complete the following information):**
 - OS: Ubuntu Server (where budibase is hosted)
 - Browser: Chrome, Firefox, Opera
 - Version Latest on all of them

Additional context:
The server is a home server located at my boss office which is using cloudflared tunnel to expose the port 10000 for external usage. Nevertheless the problem only happens with the mentioned browsers, it doesn't break with safari on desktop or any mobile browser.

**Comment by melohagan** [comment]
Hey @Cobancho 

I was able to reproduce the problem you are having, however I'm going to migrate this to a discussion, because the reproduction steps are specific to your app, and not Budibase specific insofar as I can tell.

When you select 'cambio' it shows a container that contains many fields, containers, etc. Many of which have JS with various calculations. 

You'll need to do a bit more digging on this one and find a more precise problem before we can help!
